lib/cc_plugin_imos/cc_plugin_imos/tests/test_imos.py

```diff
@@ -1,14 +1,13 @@
 #!/usr/bin/env python
 
 from cc_plugin_imos.imos import IMOS1_3Check, IMOS1_4Check
+from cc_plugin_imos.srs import  IMOSSRSBaseCheck
```

Comment:
I sure can. But to mention that in that case, it also would make sense to put the class TestUtils in a different file